app/src/pages/playground/PlaygroundDatasetExamplesTable.tsx

```diff
@@ -411,6 +413,8 @@ function TableBody<T>({ table }: { table: Table<T> }) {
               <td
                 key={cell.id}
                 style={{
+                  padding: 0,
+                  height: 1,
```

Comment:
The cell still grows to fit, we just need _some_ height declared so that `height: 100%` in the children works

Reply:
Can you put a comment to that effect?